frontend/common/src/components/Link/ExternalLink.tsx

```diff
@@ -0,0 +1,47 @@
+import React from "react";
+import { ExternalLinkIcon } from "@heroicons/react/outline";
+import { useIntl } from "react-intl";
+
+export interface ExternalLinkProps
+  extends React.LinkHTMLAttributes<HTMLAnchorElement> {
+  newTab?: boolean;
+}
+
+const ExternalLink: React.FC<ExternalLinkProps> = ({
+  newTab,
+  children,
+  ...rest
+}) => {
+  const intl = useIntl();
+
+  return (
+    <a
+      {...(newTab && {
+        target: "_blank",
```

Comment:
Sorry, went down a rabbit hole on our requirements (or not) for IE11 support 🙃 

I think we're inconsistent on support so far, but I suspect we might be obligated to offer it :/ And if so, then `rel` might be helpful to have here. re: https://techstacker.com/html-noopener-attribute/#alternative-1-use-noreferrer

Reply:
tl;dr - CRA (big stakeholders for us) might need IE11 support. https://github.com/GCTC-NTGC/gc-digital-talent/issues/2361#issuecomment-1162223234

Reply:
Hmm, I added it in [add rel attr to external link](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3130/commits/8ae3f73bb915670bc726ec2618900589adf82bb8). However, we explicitly [do not support ie11 in our `.browserslistrc`](https://github.com/GCTC-NTGC/gc-digital-talent/blob/7d22aa638eccdd9a62c7d377a42ebcbf263003f4/frontend/talentsearch/.browserslistrc#L5). It is [EOL as of the June 15, 2022](https://docs.microsoft.com/en-us/lifecycle/products/internet-explorer-11). So, we should get that clarified to see if this is something we should be supporting,

Reply:
> Sorry, went down a rabbit hole on our requirements (or not) for IE11 support 🙃

I think there was a decision made by our team to not support IE awhile back since [IE is dead](https://death-to-ie11.com/) (@tristan-orourke, can you confirm or did I imagine this decision?). The GOC's information, and even information within the same government department, is inconsistent with browser requirements:

**CRA**
https://portal-portail.crp-cpr.apps.cic.gc.ca/en-US/browser-navigateur/
https://www.canada.ca/en/revenue-agency/corporate/access-online-services-safely.html#h4 links to a [resource](https://support.microsoft.com/en-us/windows/internet-explorer-downloads-d49e1f0d-571c-9a7b-d97e-be248806ca70) that doesn't allow you to download IE and states that:

> Microsoft Edge is the recommended browser by Microsoft
> Support for Internet Explorer 11 has ended on June 15, 2022. If any site you visit needs Internet Explorer 11, you can reload it with Internet Explorer mode in Microsoft Edge.

**IRCC**
https://portal-portail.crp-cpr.apps.cic.gc.ca/en-US/browser-navigateur/

> Internet Explorer (IE) is not a supported web browser. Please use a compatible browser, such as:
> Google Chrome
> Apple Safari
> Microsoft Edge
> Mozilla Firefox

https://www.cic.gc.ca/english/helpcentre/answer.asp?qnum=314&top=3

> At this time, you may use the four most common Internet browsers. They are Internet Explorer, Firefox, Safari and Chrome.

Reply:
ah oops. i am a big fan of browserslists config in its own file, but looked quickly in the wrong place

I'm very eager to ditch it. Sorry, was distracted by that issue still being open, but I'm going to ref this prior convo and close it: https://github.com/GCTC-NTGC/gc-digital-talent/pull/2307#issuecomment-1081929227


So the `rel` attr doesn't matter, we're not supporting IE11, and according to tristan in that link, IE11 hasn't worked in a long time anyhow, so nothing lost :)

Reply:
Yep, we don't support IE at all. And as of June 15, no one else should either! Hurray!